### PR TITLE
add algokit utils

### PR DESCRIPTION
**Pull request by ryanRfox**
This PR adds docs for AlgoKit Utilities and Client Generator
- [ ] Python Utils
- [ ] TypeScript Utils
- [ ] Python Client Generator
- [ ] TypeScript Client Generator

TODO:
- [ ] No source docs for Python Client Generator
- [ ] No source docs for TypeScript Client Generator
- [ ] `get-details/algokit/utils-py/overview.md` renders oddly (likely needs to be rebuilt from source)
- [ ] The Python Utils `Capabilities` section is not as complete as the TS Utils (perhaps needs updates to source)
- [ ] `mkdocs serve` warnings:
  - [ ] 'get-details/algokit/utils-py/index.md' contains a link to 'get-details/algokit/utils-py/apidocs/algokit_utils/algokit_utils.md' which is not found in the  documentation files.
  - [ ] 'get-details/algokit/utils-py/index.md' contains a link to'get-details/algokit/utils-py/apidocs/algokit_utils/algokit_utils.md' which is not found in the documentation files.
  - [ ] 'get-details/algokit/utils-ts/capabilities/amount.md' contains a link to 'get-details/algokit/src/types/amount.spec.ts' which is not found in the documentation files.
  - [ ] 'get-details/algokit/utils-ts/capabilities/app-client.md' contains a link to 'get-details/algokit/src/types/app-client.spec.ts' which is not found in the documentation files.
  - [ ] 'get-details/algokit/utils-ts/capabilities/app-deploy.md' contains a link to 'get-details/algokit/src/app-deploy.spec.ts' which is not found in the documentation files.
  - [ ] 'get-details/algokit/utils-ts/capabilities/app.md' contains a link to 'get-details/algokit/src/app.spec.ts' which is not found in the documentation files.
  - [ ] 'get-details/algokit/utils-ts/capabilities/client.md' contains a link to 'get-details/algokit/src/network-client.spec.ts' which is not found in the documentation files.
  - [ ] 'get-details/algokit/utils-ts/capabilities/indexer.md' contains a link to 'get-details/algokit/src/indexer-lookup.spec.ts' which is not found in the documentation files.
  - [ ] 'get-details/algokit/utils-ts/capabilities/transaction.md' contains a link to 'get-details/algokit/utils-ts/code/interfaces/types_algod.PendingTransactionResponse.md' which is not found in the documentation files.
  - [ ] 'get-details/algokit/utils-ts/capabilities/transfer.md' contains a link to 'get-details/algokit/src/transfer.spec.ts' which is not found in the documentation files.

### CODE REVIEW

File: docs/clis/diagcfg/metric/disable.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ diagcfg metric disable -d dataDir [flags]
 
 ```
 
-  -d, --dataDir string   Data directory
+  -d, --dataDir string   Data directory (default "/Users/ryan/node/data")
```
Comment (bbroder-algo) [inline docs/clis/diagcfg/metric/disable.md:45]: the default is /Users/ryan ?

Comment (algohodl) [reply docs/clis/diagcfg/metric/disable.md:45]: I hope not! There must be something in the script that writes that. I'm not sure where, nor why. We don't have this default path displayed for most commands, so seems odd to include it anywhere.

**Comment by aorumbayev** [comment]
Algokit utils ts has been bumped to 3.0.0 can someone re run the script to see if typedoc errors still in-place? If yes, can we exclude algokit utils ts for the time being and merge separately later? If that's the only blocking item to get this PR merged its still tremendously gonna help if at least remaining algokit docs are merged, i assume this will allow kappa to get indexed on latest algokit docs and can generally improve devex a lot. 

cc (@Loedn @ryanRfox)
fyi (@robdmoore)

**Comment by ryanRfox** [comment]
going to close this one and open a new one based on the **algokit-cli** reorg in #1182

**Pull request closed without merging**